### PR TITLE
Add GameVault and CC-BY-NC-SA-4.0 license

### PR DESCRIPTION
**Pull request by Alfagun74**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/gamevault.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - Javascript
+  - Nodejs
+  - Docker
+tags:
+  - Miscellaneous
+depends_3rdparty: false
+demo_url: "https://demo.gamevau.lt"
+stargazers_count: 94
+updated_at: '2023-12-20'
+archived: false
```
Comment (kokomo123) [inline software/gamevault.yml:17]:
```suggestion
stargazers_count: 94
updated_at: '2023-12-20'
archived: false
```
No need to add these lines, as they will be added automatically. Oops

Comment (Alfagun74) [reply software/gamevault.yml]: thank you, i didn't know :)

**Review by nodiscc — CHANGES_REQUESTED** [review]
Since `CC-BY-NC-SA-4.0` is a non-Free license, it must be added to the `include_licenses/exclude_licenses` lists in https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.hecat/export-markdown.yml and https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.hecat/export-html.yml

**Comment by Alfagun74** [comment]
@nodiscc Thanks for informing me. I was not aware the .hecat files. Just changed it.

File: software/gamevault.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "GameVault"
+website_url: "https://gamevau.lt"
+source_code_url: "https://github.com/Phalcode/gamevault-backend"
+description: "GameVault is a gaming platform for organizing, downloading, and playing DRM-free games from your own server, complete with metadata enrichment and user-friendly interface."
+licenses:
+  - CC-BY-NC-SA-4.0
+platforms:
+  - Javascript
+  - Nodejs
+  - Docker
```
Comment (Ki-er) [inline software/gamevault.yml:10]: Js isnt needed as Nodejs is already present

```suggestion
platforms:
  - Nodejs
  - Docker
```

File: software/gamevault.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: "GameVault"
+website_url: "https://gamevau.lt"
+source_code_url: "https://github.com/Phalcode/gamevault-backend"
+description: "GameVault is a gaming platform for organizing, downloading, and playing DRM-free games from your own server, complete with metadata enrichment and user-friendly interface."
```
Comment (Ki-er) [inline software/gamevault.yml:4]:
```suggestion
description: "Organize, download, and play DRM-free games from your own server, complete with metadata enrichment and user-friendly interface."
```

File: software/gamevault.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+licenses:
+  - CC-BY-NC-SA-4.0
+platforms:
+  - Javascript
+  - Nodejs
+  - Docker
+tags:
+  - Miscellaneous
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/gamevault.yml:13]:
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes could be made.

**Comment by Alfagun74** [comment]
@Ki-er thank you for the suggestions

**Review by kokomo123 — APPROVED** [review]
As you've made the corrections, I shall approve this PR! 🎉

**Pull request merged**